src/microbe_stage/editor/upgrades/ChemoreceptorUpgradeGUI.cs

```diff
@@ -1,124 +1,241 @@
-﻿using System.Collections.Generic;
-using System.Linq;
+﻿using System;
+using System.Collections.Generic;
 using Godot;
 
 public class ChemoreceptorUpgradeGUI : VBoxContainer, IOrganelleUpgrader
 {
     [Export]
-    public NodePath? CompoundsPath;
+    public NodePath? TargetTypesPath;
```

Comment:
I'd like these added controls to have slightly more expressive names. Right now this is just "target types", which I'd expect to be a list of target types, when in fact this is the target type selection. So I suggest the name `TargetTypeSelectorPath` and renaming the related variables.